### PR TITLE
Fix mxnet build

### PR DESCRIPTION
**Pull request by Philmod**
Make sure roxygen2 is properly installed, otherwise mxnet build fails
Upgrade mxnet version.
Remove bayesCL which is not in CRAN anymore.

https://b.corp.google.com/issues/151108974

### CODE REVIEW

**Comment by Philmod** [comment]
It fixed the CPU build, trying to fix the GPU one now.

**Comment by Philmod** [comment]
Build passed.

File: gpu.Dockerfile [hunk 1]
```diff
@@ -56,7 +56,6 @@ RUN apt-get install -y --no-install-recommends ocl-icd-opencl-dev && \
 # Install GPU specific packages
 RUN CPATH=/usr/local/cuda/targets/x86_64-linux/include install2.r --error --ncpus $ncpus --repo http://cran.rstudio.com \
     h2o4gpu \
-    bayesCL \
```
Comment (djherbis) [inline gpu.Dockerfile]: Any reason we're removing this package?

Comment (Philmod) [reply gpu.Dockerfile]: See the previous comment and link to discussion.

**Pull request merged**